index.html

```diff
@@ -5155,6 +5155,56 @@ <h2>The <code>application/credential+ld+json</code> Media Type</h2>
 
 </section>
 
+<section id="presentation-media-type">
+  <h2>The <code>application/presentation+ld+json</code> Media Type</h2>
```

Comment:
```suggestion
  <h2>The <code>application/presentation+ld+json</code> Media Type</h2>
```

I presume `application/verifiable-presentation+SOMETHING` is next? :)